lib/commands/web.js

```diff
@@ -387,13 +388,18 @@ helpers.useAtomsElement = function (el) {
 helpers.convertElementsForAtoms = function (args = []) {
   let newArgs = [];
   for (let arg of args) {
-    if (!_.isNull(arg) && !_.isUndefined(arg.ELEMENT)) {
-      let atomsElement = this.getAtomsElement(arg.ELEMENT);
-      if (atomsElement === null) {
-        throw new errors.UnknownError(`Error converting element ID for using in WD atoms: '${arg.ELEMENT}'`);
+    let hasElementKey = false;
+    for (let elementKey of ['ELEMENT', util.W3C_WEB_ELEMENT_IDENTIFIER]) {
+      if (!_.isNull(arg) && !_.isUndefined(arg[elementKey])) {
+        hasElementKey = true;
+        let atomsElement = this.getAtomsElement(arg[elementKey]);
+        if (atomsElement === null) {
+          throw new errors.UnknownError(`Error converting element ID for using in WD atoms: '${arg[elementKey]}'`);
+        }
       }
-      newArgs.push(atomsElement);
-    } else {
+    }
+
+    if (!hasElementKey) {
```

Comment:
i still don't think this will work. as it stands, the code will not add args to the args list which are _not_ elements. we still want everything that was in `args` to be in `newArgs`, it's just that elements should be converted along the way.

this is also doing nothing with `atomsElement`, which is the thing we want to add to `newArgs`

i think a better solution would involve a `break` in the inner loop to ensure we only push one converted element.

Reply:
I'll write a test for this.